iron-doc-behavior.html

```diff
@@ -106,7 +108,8 @@
       },
 
       _noneToShow: function(showProtected, showInherited, descriptor, name) {
-        var items = descriptor[name];
+        var items = name === 'behaviors' ?
```

Comment:
bit: I think we're pretty consistent about putting parens around ternary conditionals